PowerShell/ScubaGear/Rego/AADConfig.rego

```diff
@@ -444,15 +444,16 @@ tests contains {
 ManagedDeviceAuth contains CAPolicy.DisplayName if {
     some CAPolicy in input.conditional_access_policies
 
-    Contains(CAPolicy.Conditions.Users.IncludeUsers, "All") == true
-    Contains(CAPolicy.Conditions.Applications.IncludeApplications, "All") == true
-    CAPolicy.State == "enabled"
+    PolicyConditionsMatch(CAPolicy) == true
```

Comment:
`PolicyConditionsMatch` has an additional check that `ExcludeRoles` is empty. Is this intentional?